Can't start game at  https://util.in:8096

Made a room with 5 people at this website and started the round, but no cards are being dealt.

Adding server logs
```
Exception in thread "Thread-237860" java.lang.RuntimeException: java.io.IOException: Connection remotely closed for 29c3f31e-0208-4e28-ab93-8cf9c889382d
        at org.atmosphere.cpr.AtmosphereResponseImpl.write(AtmosphereResponseImpl.java:759)
        at org.atmosphere.cpr.AtmosphereResponseImpl.write(AtmosphereResponseImpl.java:724)
        at org.atmosphere.cpr.AtmosphereResourceImpl.write(AtmosphereResourceImpl.java:587)
        at io.github.ytung.tractor.TractorRoom.sendSync(TractorRoom.java:519)
        at io.github.ytung.tractor.TractorRoom.access$100(TractorRoom.java:80)
        at io.github.ytung.tractor.TractorRoom$1.run(TractorRoom.java:414)
Caused by: java.io.IOException: Connection remotely closed for 29c3f31e-0208-4e28-ab93-8cf9c889382d
        at org.atmosphere.websocket.WebSocket.write(WebSocket.java:240)
        at org.atmosphere.websocket.WebSocket.write(WebSocket.java:230)
        at org.atmosphere.websocket.WebSocket.write(WebSocket.java:47)
        at org.atmosphere.cpr.AtmosphereResponseImpl$Stream.write(AtmosphereResponseImpl.java:957)
        at org.atmosphere.cpr.AtmosphereResponseImpl.write(AtmosphereResponseImpl.java:749)
        ... 5 more
```

I'm not sure why this is happening, since the broadcaster used here should only be sending messages to connected clients. Can make this entire code path resilient to such IO failures though - right now the entire thread that deals cards dies.

--- a/src/main/java/io/github/ytung/tractor/TractorRoom.java
+++ b/src/main/java/io/github/ytung/tractor/TractorRoom.java
@@ -516,7 +516,11 @@ public class TractorRoom {
 
     private void sendSync(String playerId, Broadcaster broadcaster, OutgoingMessage message) {
         if (humanControllers.containsKey(playerId))
-            humanControllers.get(playerId).write(JacksonEncoder.INSTANCE.encode(message));
+            try {
+                humanControllers.get(playerId).write(JacksonEncoder.INSTANCE.encode(message));
+            } catch (RuntimeException e) {
+                // client disconnected, ignore
+            }
         else if (aiControllers.containsKey(playerId))
             aiControllers.get(playerId).processMessage(game, message, inputMessage -> handleGameMessage(playerId, broadcaster, inputMessage));
     }

